expressible/src/main/java/panda/std/Option.java

```diff
@@ -178,6 +178,14 @@ public Option<T> peek(Consumer<T> consumer) {
         return this;
     }
 
+    public Option<T> peekIf(Consumer<T> consumer, Predicate<T> predicate) {
+        if (isDefined() && predicate.test(value)) {
+            consumer.accept(value);
+        }
+
+        return this;
```

Comment:
Predicate should be the first parameter in this method. Also, you can simplify this to:
```return peek(value -> if (predicate.test(value)) consumer.accept(value));```